pkg/teleporter/relayer.go

```diff
@@ -29,9 +29,11 @@ import (
 )
 
 const (
-	localRelayerSetupTime     = 2 * time.Second
-	localRelayerCheckPoolTime = 100 * time.Millisecond
-	localRelayerCheckTimeout  = 3 * time.Second
+	localRelayerSetupTime            = 2 * time.Second
+	localRelayerCheckPoolTime        = 100 * time.Millisecond
+	localRelayerCheckTimeout         = 3 * time.Second
+	AWMRElayerDBWriteIntervalSeconds = 10
```

Comment:
Looks like capitalization typo here.

Also, do these new consts need to be exported?